paper-progress.html

```diff
@@ -27,98 +28,179 @@
 
 To change the active progress bar color:
 
-    paper-progress::shadow #activeProgress {
-      background-color: #e91e63;
+    paper-progress {
+       --paper-progress-active-color: #e91e63;
     }
 
 To change the secondary progress bar color:
 
-    paper-progress::shadow #secondaryProgress {
-      background-color: #f8bbd0;
+    paper-progress {
+      --paper-progress-secondary-color: #f8bbd0;
     }
-    
+
 To change the progress bar background color:
 
-    paper-progress::shadow #progressContainer {
-      background-color: #64ffda;
+    paper-progress {
+      --paper-progress-container-color: #64ffda;
     }
 
 @group Paper Elements
 @element paper-progress
-@extends core-range
 @homepage github.io
 -->
 
-<link rel="import" href="../core-range/core-range.html">
+<link rel="import" href="../paper-styles/paper-styles.html">
+<link rel="import" href="../iron-range-behavior/iron-range-behavior.html">
+<link rel="import" href="../iron-flex-layout/classes/iron-flex-layout.html">
+
+<style is="custom-style">
+  * {
+    --paper-progress-active-color: var(--google-green-500);
+    --paper-progress-secondary-color: var(--google-red-100);
+    --paper-progress-container-color: var(--google-grey-300);
```

Comment:
I wonder if these should be integrated into https://github.com/PolymerElements/paper-styles/blob/master/default-theme.html somehow..